setup-server-os

```diff
@@ -129,6 +131,9 @@ elif 'system-gentoo' in cfg.modules:
 
     if 'nbd' in cfg.modules:
         install_folder('overlays/server-os-dracut-nbd')
+        if 'system-arch-dracut' in cfg.modules:
+            install_folder('overlays/server-os-dracut-arch')
+        command('dracut', '/boot/initramfs-stiefel.img', '--add', ' nbd ', '--no-hostonly', '--force')
```

Comment:
the initramfs should not be generated upon server os setup

Reply:
Why not? Otherwise the user has to generate it manually.

But we probably should use cfg.boot.load.initrd as the target file.

Reply:
the regular system initrd should support nbd then, having it two times just wastes space